app/views/keywords/_card_search_progress.html.erb

```diff
@@ -0,0 +1,25 @@
+<div id="cardSearchProgress" class="card-search-progress">
```

Comment:
As we already have a class attribute, do we still need `id="cardSearchProgress"`? If no, I think we can remove it

Reply:
Yes @junan, Turbo-Stream `update` and `replace` methods needs an identified html element (with an id) so that it knows which element to replace/update. 

We use the replace method in `search_progress_job`, via Turbo Channel helpers:
```ruby
SearchProgressChannel.broadcast_replace_to ...
```